src/raft.rs

```diff
@@ -1121,15 +1135,14 @@ impl<T: Storage> Raft<T> {
                     (m.get_msg_type() == MessageType::MsgRequestPreVote && m.term > self.term);
                 // ...and we believe the candidate is up to date.
                 if can_vote && self.raft_log.is_up_to_date(m.index, m.log_term) {
-                    // When responding to Msg{Pre,}Vote messages we include the term
-                    // from the message, not the local term. To see why consider the
-                    // case where a single node was previously partitioned away and
-                    // it's local term is now of date. If we include the local term
-                    // (recall that for pre-votes we don't update the local term), the
-                    // (pre-)campaigning node on the other end will proceed to ignore
-                    // the message (it ignores all out of date messages).
-                    // The term in the original message and current local term are the
-                    // same in the case of regular votes, but different for pre-votes.
+                    // When responding to MsgReq{Pre}Vote, we include the term from the
```

Comment:
"," is required.

Reply:
May I know why? Is these comment semantically parsed? I thought only the comments start with /// are used in docs..